lib/aws/s3.sh

```diff
@@ -172,6 +169,32 @@ s3_ls () {
     return $?
 }
 
+s3_mv () {
```

Comment:
Do we want to do a `[[ $# -lt 3 ]]` check here?

Reply:
That is a really good point, but at the point where the library is used we have already assessed that we are being run from a script and are using `set -eu` see: https://github.com/communityds/moonshell/blob/master/moon.sh#L90..L99

I am definitely a fan of input sanitisation, but for function that are not easily usable by a user by default I'm less concerned. It's why there is a wrapper script in bin/ instead.